services/actions/notifier_helper.go

```diff
@@ -528,7 +528,11 @@ func DetectAndHandleSchedules(ctx context.Context, repo *repo_model.Repository)
 	// Here we use the commit author as the Doer of the notifyInput
 	commitUser, err := user_model.GetUserByEmail(ctx, commit.Author.Email)
 	if err != nil {
-		return fmt.Errorf("get user by email: %w", err)
+		if user_model.IsErrUserNotExist(err) {
+			commitUser = user_model.NewReplaceUser(commit.Author.Name)
```

Comment:
<img width="423" alt="image" src="https://github.com/go-gitea/gitea/assets/81045/3aeeb76e-cfda-49d6-85a6-4a4fe8f9fe2a">
Then the TriggerUserID will become 0 and will be stored into database?

Reply:
yes, 0 will be stored into database.